core/store/key_store.go

```diff
@@ -7,13 +7,19 @@ import (
 
 	"github.com/ethereum/go-ethereum/accounts"
 	"github.com/ethereum/go-ethereum/accounts/keystore"
+	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/smartcontractkit/chainlink/core/logger"
 	"github.com/smartcontractkit/chainlink/core/store/models"
 	"github.com/smartcontractkit/chainlink/core/utils"
 	"go.uber.org/multierr"
 )
 
+// EthereumMessageHashPrefix is a Geth-originating message prefix that seeks to
+// prevent arbitrary message data to be representable as a valid Ethereum transaction
+// For more information, see: https://github.com/ethereum/go-ethereum/issues/3731
+const EthereumMessageHashPrefix = "\x19Ethereum Signed Message:\n32"
```

Comment:
I just wanted to minimize the amount of dynamic-ness here. Also, I named it `MessageHash` instead of `Message` to indicate that it is for 32 bytes messages strictly. LMK if you still want this changed.

Reply:
Sounds good, no need to over complicate.